### DISCUSSION TITLE
how to build a pbpk model of biologics other than mAbs?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1487] [Questions & Problems] [answered]
Hi! 
I have read several published articles about PBPK models of macromolecular drugs and found them mostly about mAbs. I am wondering if it is meaningful of building PBPK models of other macromolecular drugs like injectable insulin or GLP1 analogs？And how to build pbpk models of these biologics? Is there any articles or tutorials that I can learn from?
Looking forward for any reply. Thanks~

**Comment by PavelBal** [comment]
Hi,

sure, you can build PBPK models for any type of macromolecules. Here are examples for modeling endogenous hormones like insulin (including sc injectable) and incretins (GLP-1, GIP).

https://github.com/Open-Systems-Pharmacology/Glucose-Insulin-Model
https://github.com/Open-Systems-Pharmacology/Incretins-Model

**Reply by Gre-ux** [reply, marked as answer]
Thanks very much！
